lib/src/scrollbar.dart

```diff
@@ -0,0 +1,245 @@
+import 'dart:math' as math;
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+/// Simulates the user interacting with a Scrollbar.
+extension ScrollbarInteractions on WidgetTester {
+  /// Drag the scrollbar by [delta] pixels.
+  ///
+  /// A positive [delta] scrolls down or right, depending on the scrollbar's orientation,
+  /// and a negative [delta] scrolls up or left.
+  ///
+  /// By default, this method expects a single [Scrollbar] in the widget tree and
+  /// finds it `byType`. To specify one [Scrollbar] among many, pass a [finder].
+  Future<void> dragScrollbar(double delta, [Finder? finder]) async {
```

Comment:
We should make this action even more clear by adding the direction to the method name, e.g., `dragScrollbarDown(300)` and `dragScrollbarUp(300)`

Reply:
Updated.